### PR TITLE
support release: build debian package with version like: 1.2.3-20111012UTC-b2f2da(nigthly release) or 1.2.3(official release)

### PR DESCRIPTION
**Pull request by PengTian0**
Build debian package with version naming:
 1. for nightly build: on-imagebuilder will use version  1.2.3-commit date-commit hash, such as: 
     1.2.3-20111012UTC-b2f2da
2. for release build: on-imagebuilder will use version 1.2.3 which comes from environment variable.

### CODE REVIEW

File: HWIMO-BUILD [hunk 1]
```diff
@@ -28,19 +31,31 @@ sudo cp $output_path/builds/* common/
...
+    DATE_STRING=$(date -d "$GIT_COMMIT_DATE" -u +"%Y%m%d%H%M%SZ")
+
+    GIT_COMMIT_HASH=$(git show -s --pretty="format:%h")
+
+    CHANGELOG_VERSION=$(dpkg-parsechangelog --show-field Version)
+
+    PKG_VERSION="$CHANGELOG_VERSION-$DATE_STRING-$GIT_COMMIT_HASH"
+fi
+
+if [[ $PKG_VERSION =~ ^[0-9.]+$ ]];then
```
Comment (cgx027) [inline HWIMO-BUILD]: seems 1.2.3.4 will also match the regex, is this expected?

**Review by cgx027 — APPROVED** [review]
:+1:  with minor comment.

File: HWIMO-BUILD (hunk 1 above)
Comment (PengTian0) [reply HWIMO-BUILD]: Thanks for your comment!
I will fix it.

Comment (PengTian0) [reply HWIMO-BUILD]: fixed.

**Review by cgx027 — APPROVED** [review]

**Review by yyscamper — APPROVED** [review]

**Pull request merged**